apps/api/src/domain/models/audio-visual/video/commands/translate-video-name/video-name-translated.event.ts

```diff
@@ -34,8 +33,4 @@ const testEventId = buildDummyUuid(4);
 @CoscradEvent('VIDEO_NAME_TRANSLATED')
 export class VideoNameTranslated extends BaseEvent<VideoNameTranslatedPayload> {
     readonly type = 'VIDEO_NAME_TRANSLATED';
-
-    public static fromDto(dto: DTO<VideoNameTranslated>) {
```

Comment:
We need to do the same for other inheritance hierarchies (e.g. domain models, view models).